README.md

```diff
@@ -4,27 +4,55 @@ This repository contains all relevant project necessary to have [polkadot](https
 
 This is WIP and will evolve frequently.
 
-## How to use `pallet-chainlink`?
-
-Complete documentation is accessible in the pallet [README](pallet-chainlink/README.md).
+## What is Polkadot and Chainlink?
 
-See the full [example](substrate-node-example/runtime/src/example.rs) for more details.
+[Chainlink](https://chain.link/) is a decentralized oracle technology that allows off-chain data to be connected to any chain.
+[Polkadot](https://polkadot.network/) is a scalable, heterogeneous, multi-chain technology, that can be used to create, maintain, and connect blockchains.
 
 ## Run the example
 
+### Requirements
+
+[Rust](https://www.rust-lang.org/tools/install)
+[Docker](https://docs.docker.com/get-docker/)
+[jq](https://stedolan.github.io/jq/download/)
+
 `rust-toolchain` is used to make sure the correct rust version is used. Make sure to install the WASM target using:
 
 ```
 rustup target add wasm32-unknown-unknown
 ```
 
-`substrate-node-example` shows off out to use `pallet-chainlink` end-to-end.
+### Running the example
+
+[You can view a youtube demo here.](https://www.youtube.com/watch?v=0rZghy0TIOQ&feature=emb_title)
+
+`substrate-node-example` shows off out to use `pallet-chainlink` end-to-end. It first makes a local dummy substrate built parachain, and then spins up some local Chainlink nodes that have parameters to connect to the parachain.
 To test:
 
-* start the chain using `make run-chain`
-* start the frontend using `make run-front-end`
+- start the chain using `make run-chain`
+- start the frontend using `make run-front-end`
+
+To spin up some simple Chainlink nodes to intereact with the parachain you just created, run the setupcommand:
+`cd substrate-chainlink`
+`./setup`
+
+This will run a number of docker commands that will spin up your Chainlink node. You can access them by going to your browser at:
+`http://localhost:6691/`
+You can see in the logs of `./setup` if the address has moved.
+
+_note_
+If you'd like to restart you will have to remove all the docker containers. You can do this by running:
 
-You are now ready to send test requests and see the result being provided back by an Oracle.
+```
+docker kill substrate-chainlink_external-initiator-node3_1 substrate-chainlink_external-initiator-node2_1 substrate-chainlink_external-initiator-node1_1 substrate-chainlink_chainlink-node1_1 substrate-chainlink_chainlink-node3_1 substrate-chainlink_chainlink-node2_1 substrate-chainlink_postgres-node1_1 substrate-chainlink_substrate-adapter1_1 substrate-chainlink_substrate-adapter3_1 substrate-chainlink_postgres-node2_1 substrate-chainlink_postgres-node3_1 substrate-chainlink_substrate-adapter2_1
+
+docker system prune
+
+docker volume rm substrate-chainlink_cl1 substrate-chainlink_cl2 substrate-chainlink_cl3 substrate-chainlink_pg1 substrate-chainlink_pg2 substrate-chainlink_pg3
```

Comment:
```suggestion
docker-compose rm
```

Untested, but should work.

Reply:
Just tested, you need to use the "volume rm" command (or at least the `docker-compose down` command did not work)